package.json

```diff
@@ -6,6 +6,13 @@
     "name": "Eric Elliott",
     "url": "https://ericelliottjs.com"
   },
+  "contributors": [
```

Comment:
@ericelliott FYI, this is adding the contributors field. Personally I think it's a waste of space in `package.json` since all the history that matters is in GitHub anyway...

Reply:
Agreed

Reply:
Fixed in cb903e2.